lib/support/chromePerflogParser.js

```diff
@@ -465,8 +466,10 @@ module.exports = {
         case 'Network.eventSourceMessageReceived':
           // ignore
           break;
-        case 'Network.resourceChangedPriority':
-          // skip those for now, maybe we can do something cool with this in the future
+        case 'Network.resourceChangedPriority': {
+            let entry = entries.find((entry) => entry._requestId === params.requestId);
+            entry.newPriority = message.params.newPriority;
```

Comment:
Shouldn't this be entry.request._newPriority = params.newPriority? I don't see a newPriority property on the event elsewhere.